migrating_from_ocp_3_to_4/premigration-checklists-3-4.adoc

```diff
@@ -66,6 +66,7 @@ $ oc get csr -A | grep pending -i
 ----
 
 * [ ] The link:https://docs.openshift.com/container-platform/3.11/install_config/configuring_authentication.html#overview[identity provider] is working.
+* [ ] You have set the value of the `openshift.io/host.generated` annotation parameter to `true` for each {product-title} route to update their host name for the target cluster. Otherwise, the migrated routes retain the source cluster host name.
```

Comment:
@JoeAldinger Thank you for the review!
I am not sure I understand why you're suggesting changing 'their' to 'the'. It's about updating the host name of the routes, so why not 'their"? 
Or how about "...for each {product-title} route to update the routes' host name..."?

Reply:
@sbeskin-redhat Apologies this was missed in the initial review. I would like to see this slightly changed for readability (and I like your suggestion above). 

```suggestion
* [ ] You have set the value of the `openshift.io/host.generated` annotation parameter to `true` for each {product-title} route, which updates the host name of the route for the target cluster. Otherwise, the migrated routes retain the source cluster host name.
```
WDYT?

Reply:
@stevsmit 
OK, changed